src/databases/Curve2D/Curve2DCommonPluginInfo.C

```diff
@@ -92,19 +93,4 @@ Curve2DCommonPluginInfo::GetWriteOptions() const
 {
     return GetCurve2DWriteOptions();
 }
-// ****************************************************************************
```

Comment:
I've been noticing that some of these files have the GetLicense method added while others have them removed. Is there a particular reason for this? I don't really know what the value of this method is, so I'm mainly just curious.

Reply:
All of the xxxPluginInfo.h and xxxCommonPluginInfo.C files should have GetLicense removed (except for FieldViewXDB, the only one that requires it).  See #4216.  
I added logic so that only those DB plugins that define 'haslicense=true' in their xml file should have the GetLicense methods.